File: golang/patterns/behavioral/behavioral.go
package behavioral

import (
	"fmt"

	"github.com/Tamplier2911/gof-design-patterns/golang/patterns/behavioral/command"
	"github.com/Tamplier2911/gof-design-patterns/golang/patterns/behavioral/cor"
	"github.com/Tamplier2911/gof-design-patterns/golang/patterns/behavioral/interpreter"
)

func Behavioral() {
	fmt.Println("\nBehavioral")

	cor.ChainOfResponsibility()
	command.Command()
	interpreter.Interpreter()
}

File: csharp/pattenrs/Behavioral.cs
namespace Behavioral
{
    ///
    /// <summary>
    /// Class <c>BehavioralPatterns</c> invokes all Behavioral patterns for testing purposes.
    /// Behavioral patterns: are all different, and have no certain theme
    /// </summary>
    ///
    class BehavioralPatterns
    {
        public static void Run()
        {
            Console.WriteLine("\nBehavioral");

            ChainOfResponsibility.Main.Run();
            Command.Main.Run();
            Interpreter.Main.Run();
        }
    }
}
